src/pickers/TaskExecutorsPicker.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import {
+  useTranslations, Autocomplete, useModulesManager, useGraphqlQuery,
+} from '@openimis/fe-core';
+
+function TaskExecutorsPicker({
+  multiple = true,
+  withLabel,
+  onChange,
+  value,
+  readOnly,
+  required,
+}) {
+  const [searchString, setSearchString] = useState();
+  const modulesManager = useModulesManager();
+  const { formatMessage } = useTranslations('tasksManagement', modulesManager);
+
+  const { isLoading, data, error } = useGraphqlQuery(
+    `
+    query TaskExecutorsPicker($str: String) {
+      users(str: $str) {
+        edges {
+          node {
+            id
+            username
+            lastName
+          }
+        }
+      }
+    }
+  `,
+    { str: searchString },
+  );
+
+  const uniqueValues = [...new Map(value?.map((user) => [user.id, user])).values()];
```

Comment:
Why do we have to perform this operation?

Reply:
I used other query here - I was receiving all task executors which were assigned to the groups - it was possible to get duplicates. That's why I implemented this. Since we're not using this query, I removed it and I'm using _value_.